src/index.ts

```diff
@@ -5,6 +5,9 @@ import "./types/complex-types/complex-type"
 export {
     types
 } from "./types"
+export {
+    IType
```

Comment:
Unfortunately "./types" does not exports IType, so should be the same as requiring it here :)